command/agent/cert_end_to_end_test.go

```diff
@@ -301,6 +282,11 @@ func testCertEndToEnd(t *testing.T, withCertRoleName, ahWrapping bool) {
 		}
 	}
 	checkToken()
+
+	err = <-errchan
```

Comment:
The tests are passing, but for the wrong reasons. The tests are getting context canceled after their 30s timeout when ran locally (taking ~33s per case), but we don't report this as a test failure. The same tests on the base branch takes about 4s per test case.